deploy-service/teletraanservice/src/main/java/com/pinterest/teletraan/resource/Pings.java

```diff
@@ -63,7 +63,7 @@ public Pings(@Context TeletraanServiceContext context) {
             notes = "Returns a deploy goal object given a ping request object",
             response = PingResponseBean.class)
     @RolesAllowed(TeletraanPrincipalRole.Names.PINGER)
-    @ResourceAuthZInfo(type = AuthZResource.Type.GROUP)
+    @ResourceAuthZInfo(type = AuthZResource.Type.SYSTEM)
```

Comment:
why changed from group to system?

Reply:
It was SYSTEM before AuthZ changes. I was thinking to use the group for finer control, but that might introduce breaking changes, so revert back to `SYSTEM`.